libraries/Microsoft.Bot.Builder.Ai.Translation/LocaleConverterMiddleware.cs

```diff
@@ -54,7 +54,7 @@ public async Task OnTurnAsync(ITurnContext context, NextDelegate next, Cancellat
                 {
                     if (!string.IsNullOrWhiteSpace(message.Text))
                     {
-                        string userLocale = await _userLocaleProperty.GetAsync(context).ConfigureAwait(false);
+                        string userLocale = await _userLocaleProperty.GetAsync(context, () => "en-us").ConfigureAwait(false);
```

Comment:
>"en-us" [](start = 94, length = 7)

turn this into "DefaultLocale" property and change the middleware constructor to allow you to define DefaultLocale.